src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -1118,12 +1149,14 @@ IVariableDeclarationStatement (2 declarations) (OperationKind.VariableDeclaratio
   IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'i1')
     Variables: Local_1: i1 As System.Int32
     Initializer: 
-      ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
+      IVariableInitializer (OperationKind.VariableInitializer) (Syntax: '= 1')
+        ILiteralExpression (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1) (Syntax: '1')
   IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration, IsInvalid) (Syntax: '')
     Variables: Local_1:  As System.Object
     Initializer: 
-      IInvalidExpression (OperationKind.InvalidExpression, Type: null, IsInvalid) (Syntax: '')
-        Children(0)
+      IVariableInitializer (OperationKind.VariableInitializer, IsInvalid, IsImplicit) (Syntax: '')
```

Comment:
>IsImplicit [](start = 74, length = 10)

I would expect the child of this node to be marked as implicit too. Does the parent node also use the same syntax?

Reply:
Added [https://github.com/dotnet/roslyn/issues/22576#issuecomment-334836659](https://github.com/dotnet/roslyn/issues/22576#issuecomment-334836659) to investigate.

---
In reply to: [143262510](https://github.com/dotnet/roslyn/pull/22546#discussion_r143262510) [](ancestors = 143262510)